es-app/src/guis/GuiMenu.cpp

```diff
@@ -332,6 +335,32 @@ void GuiMenu::openUISettings()
 			ViewController::get()->reloadAll();
 	});
 
+	// Optionally ignore leading articles when sorting game titles
+	auto ignore_articles = std::make_shared<SwitchComponent>(mWindow);
+	ignore_articles->setState(Settings::getInstance()->getBool("IgnoreLeadingArticles"));
+	s->addWithLabel("IGNORE ARTICLES (NAME SORT ONLY)", ignore_articles);
+	s->addSaveFunc([ignore_articles, window] {
+		bool articles_are_ignored = Settings::getInstance()->getBool("IgnoreLeadingArticles");
+		Settings::getInstance()->setBool("IgnoreLeadingArticles", ignore_articles->getState());
+		if (ignore_articles->getState() != articles_are_ignored)
+		{
+			//For each system...
+			for (auto it = SystemData::sSystemVector.cbegin(); it != SystemData::sSystemVector.cend(); it++)
+			{
+				//Apply sort recursively
+				FileData* root = (*it)->getRootFolder();
+				root->sort(FileSorts::SortTypes.at(Settings::getInstance()->getInt("SortType")));
```

Comment:
@PhilaPhan80 I was testing this today, and it works well but this setting should be part of the future PR, not this one, correct?

If you can fix this I'll test it again and merge it, as I have a Pi3 available for a while for these.

Apologies for the delay, and thanks for putting this together.